wordpress-mobile/WordPress-Android
# Removing custom Action Bar layout from preference activities

Fixes #3873

To test:
Go to the Site Settings, Account Settings, and App Settings screens and verify that font in the Action Bar.

Needs review: @oguzkocer
